projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comment-like.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const displayNotice = message => {
+	const headerEnd = document.querySelector( '.wp-header-end' );
+	if ( ! headerEnd ) {
+		return;
+	}
+
+	// Create notice container.
+	const notice = document.createElement( 'div' );
+	notice.className = 'notice notice-error is-dismissible';
+	notice.innerHTML = `<p>${ message }</p>`;
+
+	// Create dismiss (close) button.
+	const button = document.createElement( 'button' );
+	button.setAttribute( 'type', 'button' );
+	button.className = 'notice-dismiss';
+
+	const dismissText = window.wpcomCommentLikesData?.dismiss_notice_text ?? 'Dismiss this notice';
```

Comment:
> `??`

When would we ever fall back to the literal?

Reply:
Removed the fallback since when we load the script we're localizing the text and therefore should always exist.